pkg/ddevapp/global_dotddev_assets/commands/host/dbeaver

```diff
@@ -10,6 +10,9 @@
 if [ "${DDEV_PROJECT_STATUS}" != "running" ]; then
   echo "Project ${DDEV_PROJECT} is not running, starting it"
   ddev start
+  # run this script again with the updated environment
+  DDEV_PROJECT_STATUS=running ddev "$(basename "$0")" "$@"
```

Comment:
> If there is a problem with ddev start that results in containers not being started correctly, could this result in infinite recursion?

`DDEV_PROJECT_STATUS=running` is here to prevent an infinite loop (that's why I added this override).

It's easy to test, the code below will cause an infinite run:

```
DDEV_PROJECT_STATUS=stopped ddev "$(basename "$0")" "$@"
```

Reply:
Does DDEV respect environment variables that tell it the status of its projects? I'd expect it to set it each time from within ddev itself, based on the actual status of the project.
Is there a way to test that?